### ISSUE TITLE
Regression: ConeDecomposition / OpenFacets wrong in version >= 3.7.3

### ISSUE THREAD

**Issue by fingolfin** [Normaliz/Normaliz#249]
While working on a new version of `NormalizInterface`, I noticed the following: with Normaliz <= 3.7.2, I get this:
```
gap> M := [
> [ 1, 1, 1, 0, 0, 0 ],
> [ 1, 1, 0, 1, 0, 0 ],
> [ 1, 0, 1, 0, 1, 0 ],
> [ 1, 0, 0, 1, 0, 1 ],
> [ 1, 0, 0, 0, 1, 1 ],
> [ 0, 1, 1, 0, 0, 1 ],
> [ 0, 1, 0, 1, 1, 0 ],
> [ 0, 1, 0, 0, 1, 1 ],
> [ 0, 0, 1, 1, 1, 0 ],
> [ 0, 0, 1, 1, 0, 1 ],
> ];;
gap> cone := NmzCone(["normalization", M]);;
gap> NmzConeDecomposition(cone);
[ [ false, false, false, false, false, false ], 
  [ false, true, false, false, false, true ], 
  [ true, false, false, false, false, true ], 
  [ true, false, false, false, false, true ], 
  [ true, false, false, false, true, true ], 
  [ true, false, false, false, false, true ], 
  [ false, false, false, false, false, true ], 
  [ true, false, true, false, false, true ], 
  [ false, false, false, false, false, true ], 
  [ false, false, false, false, true, true ], 
  [ true, false, false, false, false, true ], 
  [ true, false, false, false, false, true ], 
  [ true, false, false, false, false, true ], 
  [ true, false, false, false, true, true ], 
  [ false, false, false, false, false, true ], 
  [ false, false, true, false, false, true ], 
  [ false, false, false, false, false, true ], 
  [ false, false, true, false, false, true ] ]
```
But with Normaliz 3.7.3 and 3.7.4, I get this instead:
```
gap> M := [
> [ 1, 1, 1, 0, 0, 0 ],
> [ 1, 1, 0, 1, 0, 0 ],
> [ 1, 0, 1, 0, 1, 0 ],
> [ 1, 0, 0, 1, 0, 1 ],
> [ 1, 0, 0, 0, 1, 1 ],
> [ 0, 1, 1, 0, 0, 1 ],
> [ 0, 1, 0, 1, 1, 0 ],
> [ 0, 1, 0, 0, 1, 1 ],
> [ 0, 0, 1, 1, 1, 0 ],
> [ 0, 0, 1, 1, 0, 1 ],
> ];;
gap> cone := NmzCone(["normalization", M]);;
gap> NmzConeDecomposition(cone);
[ [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ], [  ] ]
```
I have verified that `getOpenFacets()` really returns a `vector<vector<bool> >` of length 18, which contains 18 vectors of lengths 0.

**Comment by w-bruns** [comment]
Fixed in current master.

**Comment by fingolfin** [comment]
Confirmed. I look forward to 3.7.5 (or 3.8.0, or whatever the next version will be :-). Once that is out, I can update NormalizInterface to use it, and will then release that, too.

**Comment by w-bruns** [comment]
Fixed in 3.8.0
